cyclonedx_py/parser/environment.py

```diff
@@ -43,19 +44,25 @@
 else:
     from importlib_metadata import metadata, PackageMetadata as _MetadataReturn
 
-from cyclonedx.model import LicenseChoice
+from cyclonedx.model import License, LicenseChoice
 from cyclonedx.model.component import Component
 from cyclonedx.parser import BaseParser
 
 
+@enum.unique
+class LICENSE_OUTPUT_FORMAT(enum.Enum):
+    EXPRESSION = 'expression'
+    LICENSE = 'license'
+
+
 class EnvironmentParser(BaseParser):
     """
     This will look at the current Python environment and list out all installed packages.
 
     Best used when you have virtual Python environments per project.
     """
 
-    def __init__(self, use_purl_bom_ref: bool = False) -> None:
+    def __init__(self, use_purl_bom_ref: bool = False, licence_output_format: str = 'expression') -> None:
```

Comment:
:x: 
* `licence_output_format` is supposed to be not an arbitrary string, but of the type the new enum is.
* the default value should be derived from that enum
* type: `licence_output_format` -> `license_output_format`